fire_perimeter/client.py

```diff
@@ -356,9 +391,8 @@ async def main():
 
         point = shape(feature['geometry'])
 
-        yesterday = date.today() - timedelta(days=1)
-
-        await generate_data(yesterday, point, fire_number, current_size)
+        # run up to today
```

Comment:
@ashlinrichardson I used to run up to the previous day (my thinking was due to lag in data, there was no point in running up to day) - but it makes the results seem a bit weird, because the "date of interest" is always a day behind. So it "feels" like the data is stale. Also - I mean - it's not impossible that the data could be loaded faster. Right?

Reply:
Hi @Sybrand as we saw from the latency log, data are sometimes available same-day. Would need to assume it could be there to find it I guess! Haha. Also, are you considering the time zone? Wouldn't want to rule out a new acquisition that arrived 3h from when it was taken, because our clock is 8h off : ) 
Fortunate to have you to discuss these issues with, thanks again!